exporter/cat1_view_helper.go

```diff
@@ -260,3 +260,13 @@ func codeDisplayAttributeIsCodes(attribute string) bool {
 func isNil(i interface{}) bool {
 	return i == nil
 }
+
+func hasSetIncisionTime(e models.HasEntry) bool {
```

Comment:
Does it make more sense for these to be methods on the `models.HasEntry` struct? I think we're trying to minimize helper functions like this where they only act on one model.